### PR TITLE
Merge NLX 1.4.8

### LINKED ISSUE #182: "Invalid OAuth parameter: prompt" error when trying to verify fxa identity in mozillians
STR:
1. Login to http://web-mozillians-staging.production.paas.mozilla.community/
2. Navigate to settings page
3. Click "Add Identity" and select "Log in with Firefox"

Expected:
Enter credentials window is shown

Actual:
"Invalid OAuth parameter: prompt" error is shown

add fxa in stage

### PR DESCRIPTION
**Pull request by hidde**
Fixes #182 , #64

_Note: the fix for #64 is still pending QA, so this cannot release just yet_

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,10 +3,19 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.4.8]
+
+### Changed
+
+- Now respects `account_linking=true` in addition to `prompt=select_account` in order to
```
Comment (gdestuynder) [inline CHANGELOG.md]: can we call this verification? ;-)

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,10 +3,19 @@
...
+  prevent autologin when linking account.
+- Uses Histori API to add entry to history when doing autologin so that user can go back.
```
Comment (gdestuynder) [inline CHANGELOG.md]: s/histori/history/

Comment (hidde) [reply CHANGELOG.md]: fixed!

File: CHANGELOG.md (hunk 1 above)
Comment (hidde) [reply CHANGELOG.md]: changed; also updating it in Mozillians  (https://github.com/mozilla/mozillians/pull/2541)

**Pull request merged**